modules/control/update.tpl

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+sudo apt-get update
+sudo apt-get install git ansible awscli -y 
+sudo apt-get install ntp
+sudo service ntp stop
+sudo ntpd -gq
+sudo service ntp start
```

Comment:
I think it might be better to add ntp to the existing `init.tpl` within the control module. It has a few other things that might be useful.